### ISSUE TITLE
AttentionSeq2seq Shape Mismatches

### ISSUE THREAD

**Issue by babylonico** [farizrahman4u/seq2seq#57]
I have x_train.shape = (12097, 5, 1) and y_train.shape=(12097, 10, 1).

I initialize a model:
model = AttentionSeq2seq(input_dim=1, input_length=x_train.shape[1], hidden_dim=64, output_length=y_train.shape[1], output_dim=1)
model.compile(loss='mse', optimizer='rmsprop')

I try to fit: model.fit(x_train, y_train,nb_epoch=1,batch_size=16,verbose=1)

And immediately I get the error:
Shape mismatch: x has 64 cols (and 16 rows) but y has 1 rows (and 64 cols)

For some reason the batch size isn't being applied to y samples. 

Similar thing happens with Seq2seq model, where I specify input_batch_size = (16,x_train.shape[1],1), except the error comes at the end of the epoch, again having to do with the dimensionality of y.

Are my data or model initialization wrong? Worked fine with SimpleSeq2seq
